docs/documentation/server_admin/topics/sessions/preloading.adoc

```diff
@@ -9,10 +9,11 @@ Therefore, the offline sessions are lazily fetched from the database by default.
 
 However, {project_name} can be configured to preload the offline sessions from the database into the Infinispan caches during the server startup.
 It can be achieved by setting `preloadOfflineSessionsFromDatabase` property in the `userSessions` SPI to `true`.
+This functionality is currently deprecated and will be removed in a future release.
 
 The following example shows how to configure offline sessions preloading.
 
 [source,bash]
 ----
-bin/kc.[sh|bat] start --spi-user-sessions-infinispan-preload-offline-sessions-from-database=true
+bin/kc.[sh|bat] start --features-enabled --spi-user-sessions-infinispan-preload-offline-sessions-from-database=true
```

Comment:
Doesnt this miss the "offline-session-preloading" after the "--features-enabled"?

Reply:
Thank you for spotting this, see martin's comment below.